ListView-and-Adapters-HW/app/src/main/java/com/elysium/listview_and_adapters_hw/MainActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.elysium.listview_and_adapters_hw;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity {
+
+    EditText mPlayerName;
+    Button mAdd;
+    Button mRemove;
+    ListView mListView;
+    List<String> mPlayerList;
```

Comment:
This totally works if you just want a single String for each player. If you want to have a list of `Player` objects, then right here is where you'd declare `List<Player> mPlayerList` and you'd populate it with new instances of the `Player` class rather than just Strings.